code/datums/mood_events/generic_positive_events.dm

```diff
@@ -239,3 +239,8 @@
 	description = "Fishing is relaxing"
 	mood_change = 5
 	timeout = 3 MINUTES
+
+/datum/mood_event/root
+	description = span_nicegreen(I rooted recently, it feels good to charge naturally.\n")
```

Comment:
```suggestion
	description = span_nicegreen("I rooted recently, it feels good to charge naturally.\n")
```
never let me review at 2 am again

Reply:
so true. I need to unhook my email to these review comments lol